src/providers/wfs/qgswfsdataitems.cpp

```diff
@@ -335,9 +335,9 @@ QVector<QgsDataItem *> QgsWfsDataItemProvider::createDataItems( const QString &p
         for ( const QString &encodedUri : encodedUris )
         {
           QgsWFSDataSourceURI uri( encodedUri );
-          QgsDebugMsg( QStringLiteral( "WFS full uri: '%1'." ).arg( uri.uri() ) );
+          QgsDebugMsg( QStringLiteral( "WFS full uri: '%1'." ).arg( uri.uri( false ) ) );
```

Comment:
Also a candidate for message level 4? It's pure debug info, right?

Reply:
Yes, I just forgot to add the level here too, thanks for reminding me.